packages/javascript-sdk/src/fr-auth/callbacks/index.ts

```diff
@@ -22,7 +23,7 @@ class FRCallback {
   /**
    * Gets the name of this callback type.
    */
-  public getType(): string {
+  public getType(): CallbackType {
```

Comment:
Is there any side-effects from changing this? I would think this is actually not entirely correct as the function doesn't return the enum itself, but a value of the enum, yeah?